TransactionReporting/get-account-updater-job-details.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+from authorizenet import apicontractsv1
+from authorizenet.apicontrollers import getAUJobDetailsController
+from authorizenet.constants import constants
+
+def get_account_updater_job_details():
+    merchantAuth = apicontractsv1.merchantAuthenticationType()
+    merchantAuth.name = constants.apiLoginId
+    merchantAuth.transactionKey = constants.transactionKey
+
+    paging = apicontractsv1.Paging()
+    # Paging limit can be up to 1000 for this request
+    paging.limit = 1000
+    paging.offset = 2
+    request = apicontractsv1.getAUJobDetailsRequest()
+    request.merchantAuthentication = merchantAuth
+    request.paging = paging
+    request.month = "2018-08"
+    request.modifiedTypeFilter = "all"
+    request.refId = "123456"
+    controller = getAUJobDetailsController(request)
+    controller.execute()
+    response = controller.getresponse()
+
+    if response is not None:
+        if response.messages.resultCode == apicontractsv1.messageTypeEnum.Ok:
+
+                if hasattr(response, 'auDetails'):
+                    print('SUCCESS: Get Account Updater job details for Month and year :' + request.month)
+                    if response.messages is not None:
+                        print('Message Code: %s' % response.messages.message[0]['code'].text)
+                        print('Message Text: %s' % response.messages.message[0]['text'].text)
+                        print('Total Number In Results: %s' % response.totalNumInResultSet)
+                        print('\n')
+
+                        for details in response.auDetails.auDelete:
+                            print('Deleted Profile:')
+                            # auDelete Start
+                            print('Customer Payment Profile ID: %s' % details.customerPaymentProfileID)
+                            print('Customer  Profile ID: %s' % details.customerProfileID)
```

Comment:
Remove extra space in "Customer  Profile ID". Also Customer Profile ID should come before Customer Payment Profile ID.

Reply:
Removed extra space in "Customer Profile ID" and placed  Customer Profile before Customer Payment Profile ID.